.github/workflows/myfirst.yml

```diff
@@ -0,0 +1,12 @@
+name: Continuous Integration
+on: [push]
+
+jobs:
+  build:
+    name: Build and test
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
```

Comment:
Our instructions are slightly out here, so you may notice you resolve a warning if you switch to v3!
```
    - uses: actions/checkout@v3
```